build/ferretdb-bw/Dockerfile

```diff
@@ -6,10 +6,13 @@ FROM --platform=linux/amd64 ghcr.io/ferretdb/postgres-documentdb-dev:17-ferretdb
 
 RUN --mount=type=cache,target=/var/cache/apt <<EOF
 
+apt install -y curl
+
 curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | bash
 
-curl -L https://pgp.mongodb.com/server-7.0.asc | apt-key add -
-echo "deb [ arch=amd64,arm64 ] https://repo.mongodb.org/apt/debian bookworm/mongodb-org/7.0 main" | tee /etc/apt/sources.list.d/mongodb-org-7.0.list
+curl -L https://pgp.mongodb.com/server-7.0.asc | \
```

Comment:
Those commands followed https://www.mongodb.com/docs/mongodb-shell/install/, but the instructions on that page a just broken now; for example, "If you receive an error indicating that gnupg is not installed" is just not possible.

I guess we have to do our own now.